documentation/modules/exploit/windows/misc/ivanti_avalanche_mdm_bof.md

```diff
@@ -0,0 +1,60 @@
+## Vulnerable Application
+
+Ivanti Avalanche powered by Wavelink is a mobile device management system. Network security
+features allow you to manage wireless settings (including encryption and authentication),
+and apply those settings on a schedule throughout the network. Software distribution features
+allow you to schedule software distribution to specific devices from the Avalanche Console.
+Avalanche also provides tools for managing alerts and reports.
+
+This module exploits a buffer overflow condition in Ivanti Avalanche MDM versions before `v6.4.1`.
+An attacker can send a specially crafted message to the Wavelink Avalanche Manager,
+which could result in arbitrary code execution with the `NT/AUTHORITY SYSTEM` permissions.
+This vulnerability occurs during the processing of 3/5/8/100/101/102 item data types.
+The program tries to copy the item data using `qmemcopy` to a fixed size data buffer on stack.
+Upon successful exploitation the attacker gains full access to the target system.
+
+The original analysis and the vulnerability discovery is done by the Tenable.
+Check [here](https://www.tenable.com/security/research/tra-2023-27) for public advisory.
+
+## Testing
```

Comment:
I followed these steps and they worked great!

https://github.com/rapid7/metasploit-framework/blob/c4396f2c6f45da68ec3fd8749a7d86fb54c2fa57/documentation/modules/exploit/windows/http/ivanti_avalanche_filestoreconfig_upload.md?plain=1#L11-L33